### ISSUE TITLE
Accessibility Feedback for Midpoint Review [Caregivers, Facilities Search on 10-10CG)

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#27538]
# Accessibility Feedback for Midpoint Review [Caregivers, Facilities Search on 10-10CG)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Angela and Josh

## Consider

**Comment by AngelaFowler82** [comment]
I would like to add the following:
 	Headings should be nested properly. The first heading on the page should be an H1, and it should be the main heading.
•	The hint text indicating that the “City, State or Zip” input field is prepopulated based on the user’s current address should be announced by screen readers. Suggest using aria-describedby to ensure it is announced properly.

**Comment by hdjustice** [comment]
Closing ticket - The team has decided on a different design, and will be going through a new Midpoint review.
